### ISSUE TITLE
slurm submission to gpu nodes does not correctly use assigned gpu's

### ISSUE THREAD

**Issue by McHaillet** [SBC-Utrecht/PyTomPrivate#106]
Submitted job started throwing cuda allocation errors once another cluster user started a job on the same gpu node. The job apparently uses different gpu's than the ones it was assigned in slurm.

It might have to do with exporting the cuda visible devices, see sbatch header:

```
#!/usr/bin/bash
#SBATCH --time        24:00:00
#SBATCH -N 1
#SBATCH --partition gpuq
#SBATCH --ntasks-per-node 4
#SBATCH --job-name    TM_Batch_ID_0                                                                       
#SBATCH --output      /data2/mchaillet/processing/pytom_paper/HEKFWT_27-7-20E/frames/imod/pytom_deconv/LogFiles/%j-%x_tomo200528_100.out 
#SBATCH --gres=gpu:4

export CUDA_VISIBLE_DEVICES=0,1,2,3
```

**Comment by sroet** [comment]
slurm should set CUDA_VISIBLE_DEVICES on it's own if you requested GPU's (it does on our hpc setup)

**Comment by sroet** [comment]
nvm fixed by #110
